sqllogictest/src/runner.rs

```diff
@@ -33,6 +34,19 @@ pub trait AsyncDB: Send {
     async fn sleep(dur: Duration) {
         std::thread::sleep(dur);
     }
+
+    /// Hook for parallel task running.
+    async fn run_task<F>(futures: Vec<F>)
+    where
+        F: Future + Send + 'static,
+        <F as Future>::Output: Send + 'static,
+    {
+        let handles = futures.into_iter().map(|future| tokio::spawn(future));
+
+        for handle in handles {
+            handle.await.unwrap();
+        }
```

Comment:
It seems that `tokio::spawn` is not necessary here. Can be replaced by `join_all` (?